src/constants.py

```diff
@@ -9,6 +9,7 @@
 USAGE_TEXT = 'Send a search phrase or a link to play a song!'
```

Comment:
Why not use Enums instead of constants?

Reply:
My understanding of enums is its for variables that you want to enumerate. 
Unless you are talking about put all the program constants in one enum. So instead of making the files import * from constants.py, do you mean you can import one enum variable from constants.py?

Reply:
Yup, that's exactly what I am talking about!

Reply:
I never really thought of enums like that but it definitely would be better then importing * from a file. Probably will put this as a refactor issue instead of fixing it in the minor bug fixes PR.

Reply:
Issue created #46